node/coinstacks/solana/api/src/controller.ts

```diff
@@ -73,9 +79,67 @@ export class Solana implements BaseAPI {
   @Response<ValidationError>(422, 'Validation Error')
   @Response<InternalServerError>(500, 'Internal Server Error')
   @Get('account/{pubkey}/txs')
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  async getTxHistory(@Path() pubkey: string, @Query() _cursor?: string, @Query() _pageSize = 10): Promise<TxHistory> {
-    return { pubkey } as TxHistory
+  async getTxHistory(@Path() pubkey: string, @Query() cursor?: string, @Query() pageSize = 10): Promise<TxHistory> {
+    const urlParams = new URLSearchParams()
+    urlParams.append('api-key', RPC_API_KEY ?? '')
+    urlParams.append('limit', pageSize.toString())
+    if (cursor) {
+      urlParams.append('before', cursor)
+    }
+
+    try {
+      const { data } = await axiosNoRetry.get<RawTx[]>(
+        `${INDEXER_URL}/v0/addresses/${pubkey}/transactions/?${urlParams.toString()}`
```

Comment:
then use the `param` config on get to add limit and before (before can be undefined depending on the cursor without need to check anything). feels a bit cleaner

Reply:
Added in [774cde9](https://github.com/shapeshift/unchained/pull/1052/commits/774cde9191e9bca28050cd3c10aac6bd998b7721)